dashboard/app/assets/javascripts/application.js.erb

```diff
@@ -35,51 +35,3 @@
 //= require utils
 //= require_directory .
 //= require_self
-
-//= require react
-//= require_tree ./components
-
-<% environment.context_class.instance_eval { include ApplicationHelper } %>
```

Comment:
My best understanding of this line, is that it exposed helpers in `ApplicationHelper` to any embedded ruby in .erb files that got included into application.js (maybe just after the require_self?).  At this point there's no .js.erb left in dashboard except this one, so I believe the line is unnecessary.